src/cittadini/api/cittadini/v1/_openapi.yml

```diff
@@ -36,6 +36,42 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorDTO'
+  /token/oneidentity:
+    get:
+      tags:
+        - arc auth
+      summary: "Provide authorization endpoint for exchanging code for token"
+      operationId: getAuthenticationToken
+      parameters:
+        - name: code
+          in: query
+          description: "The value used for exchanging code for token"
+          required: true
+          schema:
+            type: string
+        - name: state
+          in: query
+          description: "The value of the state this value is handled by the security component"
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: "Token Response"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TokenResponse'
+        '401':
+          description: "Wrong or missing code or state"
+        '429':
+          description: "Too many Requests"
+        '500':
```

Comment:
500 does not represent the service unavailable, try 502 or 503 if you want to represents it

Reply:
changed the service unavailable to 503